tests/test_scripts/test_solph/test_tsam/test_full_load_hours_and_variable_costs.py

```diff
@@ -39,17 +41,20 @@
 logger.define_logging()
 logging.info("Initialize the energy system")
 
-tindex_original = pd.date_range("2022-01-01", periods=8, freq="H")
-tindex = pd.date_range("2022-01-01", periods=4, freq="H")
+tindex_original = pd.date_range("2022-01-01", periods=8, freq="h")
+tindex = pd.date_range("2022-01-01", periods=4, freq="h")
 
-energysystem = solph.EnergySystem(
-    timeindex=tindex,
-    tsa_parameters={
-        "timesteps_per_period": 2,
-        "order": [1, 1, 1, 0],
-    },
-    infer_last_interval=True,
-)
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", SuspiciousUsageWarning)
```

Comment:
Maybe this is nothing to be resolved in this PR, but shouldn't this be a `ExperimentalFeatureWarning` instead of a `SuspiciousUsageWarning`?